app/src/main/java/com/yml/womensafety/authentication/RegistrationActivity.kt

```diff
@@ -80,6 +73,10 @@ class RegistrationActivity : AppCompatActivity() {
                     startActivity(Intent(applicationContext, LoginActivity::class.java))
                     finish()
                 }
+                else{
+                    Toast.makeText(applicationContext, "Registration failed", Toast.LENGTH_SHORT)
```

Comment:
Why hardCoded string is here?

Reply:
Extracted it to string resource file.

Reply:
If you have extracted then why "Registration Failed" is still present?